### PR TITLE
Update AboutUs - Ang Wei Jun

### PR DESCRIPTION
**Pull request by aweijun**
Update the AboutUs.md with personal information. Uploaded a personal
photo to docs/images as aweijun.png.

This is done to introduce the developers working on the product.

### CODE REVIEW

File: docs/team/aweijun.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+title: aweijun's Project Portfolio Page
```
Comment (arcornior) [inline docs/team/aweijun.md:3]: Should your name be your full name instead of your GitHub username?

File: docs/team/johndoe.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: page
-title: John Doe's Project Portfolio Page
+title: aweijun's Project Portfolio Page
```
Comment (arcornior) [inline docs/team/johndoe.md:3]: Should we leave this file untouched first?

**Review by arcornior — APPROVED** [review]
LGTM. Do check your names

**Review by Ardentsoul — APPROVED** [review]
LGTM.

**Pull request merged**